### ISSUE TITLE
Images from Camera

### ISSUE THREAD

**Issue by ayubaisa** [Nimrodda/WizarDroid#89]
Hi,

I'm trying to display images i get from camera in the summary page of the wizard and i'm having difficulty. is there any solution you can suggest?

thanks

**Comment by Nimrodda** [comment]
Hi, how can I suggest solution if you haven't shown any code snippet of what you're trying to do :P

**Comment by Zapnologica** [comment]
Im also not to sure if the image issues related to the wizard library? Maybe reword your question to ask the specific error. For example how do I get the image url from all the steps at the end?

**Comment by ayubaisa** [comment]
Hi, how do i pass image displayed in an imageview in a page to another page? i have tried to use the byte array and the bitmap but it didnt work. see below :

`
                    Picbmp = (Bitmap) data.getExtras().get("data");
                    ByteArrayOutputStream stream = new ByteArrayOutputStream();

```
                Picbmp.compress(Bitmap.CompressFormat.PNG, 100, stream);
                byte[] PicbyteArray = stream.toByteArray();

                // convert byte array to Bitmap

                Bitmap photoBmp = BitmapFactory.decodeByteArray(PicbyteArray, 0, PicbyteArray.length);
                photoImage.setImageBitmap(photoBmp);
```

`

i want to pass the image that i display in the "photoImage" imageview or other previous pages, so i can render them in the next page

**Comment by Nimrodda** [comment]
As I commented in the duplicate issue:

Passing bitmaps between steps is a bad idea cuz it's slow and takes a lot of memory. You're on the right path with passing URIs. Currently, there is no support for arrays at all, nor URIs. What u can do for now is convert the uri to string. In case of multiple URIs u can make a comma separated string. This is kind of a hack but at least with current version that the only solution I can suggest.
